### ISSUE TITLE
Mode bug on env Berzerk-v0 with ALE 0.6

### ISSUE THREAD

**Issue by P3RI9** [openai/gym#1583]
Since ALE 0.6 update `env = gym.make("Berzerk-v0")` throws error:
```
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-1-7eaf4fc072b0> in <module>
      1 import gym
----> 2 env = gym.make("Berzerk-v0")

~/anaconda3/lib/python3.7/site-packages/gym/envs/registration.py in make(id, **kwargs)
    154 
    155 def make(id, **kwargs):
--> 156     return registry.make(id, **kwargs)
    157 
    158 def spec(id):

~/anaconda3/lib/python3.7/site-packages/gym/envs/registration.py in make(self, path, **kwargs)
     99             logger.info('Making new env: %s', path)
    100         spec = self.spec(path)
--> 101         env = spec.make(**kwargs)
    102         # We used to have people override _reset/_step rather than
    103         # reset/step. Set _gym_disable_underscore_compat = True on

~/anaconda3/lib/python3.7/site-packages/gym/envs/registration.py in make(self, **kwargs)
     71         else:
     72             cls = load(self._entry_point)
---> 73             env = cls(**_kwargs)
     74 
     75         # Make the enviroment aware of which spec it came from.

~/anaconda3/lib/python3.7/site-packages/gym/envs/atari/atari_env.py in __init__(self, game, mode, difficulty, obs_type, frameskip, repeat_action_probability, full_action_space)
     67                 repeat_action_probability)
     68 
---> 69         self.seed()
     70 
     71         self._action_set = (self.ale.getLegalActionSet() if full_action_space

~/anaconda3/lib/python3.7/site-packages/gym/envs/atari/atari_env.py in seed(self, seed)
     96         assert self.game_mode in modes, (
     97             "Invalid game mode \"{}\" for game {}.\nAvailable modes are: {}"
---> 98         ).format(self.game_mode, self.game, modes)
     99         assert self.game_difficulty in difficulties, (
    100             "Invalid game difficulty \"{}\" for game {}.\nAvailable difficulties are: {}"

AssertionError: Invalid game mode "0" for game berzerk.
Available modes are: [ 1  2  3  4  5  6  7  8  9 16 17 18]
```
Default mode on `atari_env.py` is 0. Berzerk doesn't admit this mode. Changing default mode=1 works to me because I'm only using Berzerk enviroment.

PD1: Just for curius, what is ALE?
PD2: This is my first post, sorry if I've made mistakes.

**Comment by zhefan** [comment]
Are you using gym 0.13.0? If so, perhaps try 0.12.5

**Comment by P3RI9** [comment]
Using 0.12.5 works fine but i wrote this mainly for bug fixes. By the way, thanks!

**Comment by JesseFarebro** [comment]
@P3RI9 this should be fixed in 0.13.1

**Comment by christopherhesse** [comment]
Thanks for answering this @JesseFarebro!
